scripts/run_validator_node.sh

```diff
@@ -0,0 +1,187 @@
+echo "Stop old node if it's running..."
+
+#Check status of process
+if [[ $arch == "x86_64" ]]; then
+    $sudo_cmd systemctl stop atleta-validator &>/dev/null || true
+else
+    $sudo_cmd launchctl unload /Library/LaunchDaemons/com.example.atleta-validator.plist || true
+fi
+
+echo "Download binary..."
+$sudo_cmd curl -sSL https://github.com/Atleta-network/atleta/releases/download/v1.0.0/atleta-node -o /usr/local/bin/atleta-node
+$sudo_cmd chmod +x /usr/local/bin/atleta-node
+echo "Download chain spec..."
+$sudo_cmd curl -sSL https://github.com/Atleta-network/atleta/releases/download/v1.0.0/$chain_spec-chain-spec.json -o /opt/atleta/chain_spec.json
+
+echo "Generate atleta-validator.service..."
+
+if [[ $arch == "x86_64" ]]; then
+    $sudo_cmd tee /etc/systemd/system/atleta-validator.service >/dev/null << EOF
+    [Unit]
+    Description=Atleta Validator Node Service
+    After=network.target
+
+    [Service]
+    Type=simple
+    ExecStart=/usr/local/bin/atleta-node --chain /opt/atleta/chain_spec.json --validator
+    Restart=on-failure
+    RestartSec=5m
+
+    [Install]
+    WantedBy=multi-user.target
+EOF
+    $sudo_cmd systemctl daemon-reload
+    $sudo_cmd systemctl enable atleta-validator
+    $sudo_cmd systemctl start atleta-validator
+else
+    cat << EOF | $sudo_cmd tee /Library/LaunchDaemons/com.example.atleta-validator.plist >/dev/null
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+    <dict>
+        <key>Label</key>
+        <string>com.example.atleta-validator</string>
+        <key>Program</key>
+        <string>/usr/local/bin/atleta-node</string>
+        <key>ProgramArguments</key>
+        <array>
+            <string>/usr/local/bin/atleta-node</string>
+            <string>--chain</string>
+            <string>/opt/atleta/chain_spec.json</string>
+            <string>--validator</string>
+        </array>
+        <key>StandardErrorPath</key>
+        <string>/var/log/atleta-validator.log</string>
+        <key>RunAtLoad</key>
+        <true/>
+        <key>KeepAlive</key>
+        <dict>
+            <key>SuccessfulExit</key>
+            <false/>
+        </dict>
+    </dict>
+    </plist>
+EOF
+    $sudo_cmd launchctl load /Library/LaunchDaemons/com.example.atleta-validator.plist
+    $sudo_cmd launchctl start com.example.atleta-validator
+fi
+
+echo "Atleta validator node was started successfully."
+
+if [ $keychain_exists -eq 0 ]; then
+	echo "Waiting for node to start to generate session keys (up to 5 minutes)..."
+	i=0
+	while [ -z "$session_keys" ]; do
+		echo "Waiting..."
+		if [ $i -gt 60 ]; then
+			echo
+			echo "Node didn't start after 5 minutes." >&2
+			exit 1
+		fi
+		sleep 5
+		set +e
+		session_keys=$(wscat -c ws://127.0.0.1:9944 -x '{"id":1, "jsonrpc":"2.0", "method": "author_rotateKeys", "params":[]}' 2>/dev/null | jq -r .result)
```

Comment:
please, check this works. probably, you need to run the node with `--rpc-methods=Unsafe`

Reply:
also, use curl. wscat is not so common tool

Reply:
and you need `jq` to be installed on the user's machine, so check it's installed and return the error with installation instructions if it's not.

Reply:
--rpc-methods=Unsafe is set by default. We can rotateKeys without this flag.
Replaced wscat with curl.
Added jq to doc